classes/base.lua

```diff
@@ -79,6 +79,7 @@ end
 
 function class:setOptions (options)
   options = options or {}
+  self.options.landscape = options.landscape or false
```

Comment:
```suggestion
  self.options.landscape = SU.boolean(options.landscape, false)
```

Lets use our utility function for this. This instance isn't a huge deal but it's easier to reason about when we do it the same way each time.